Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/angle_and_area_smoothing.h

```diff
@@ -329,13 +329,33 @@ void angle_and_area_smoothing(const FaceRange& faces,
   }
 }
 
-///\cond SKIP_IN_MANUAL
+/*!
+* \ingroup PMP_meshing_grp
+*
+* \short smooths a polygon mesh.
```

Comment:
Better to use the equivalent `\brief`:
- only in this file the `\short` is used (in the past as well) but nowhere else in CGAL

Reply:
fixed. Thanks for pointing out.